### PR TITLE
More investigations and mappings for HIE

### PR DESCRIPTION
**Pull request by Mwanje**
https://app.asana.com/0/1133167201254915/1201792799277852

### CODE REVIEW

**Comment by slubwama** [comment]
@Mwanje Ensure that majority of the CBC results can allow decimal values.

File: api/src/main/resources/metadata/Custom_Concepts.xml [hunk 1]
```diff
@@ -9079,7 +9178,34 @@
...
+    <concept_reference_term  concept_reference_term_id="284135"  concept_source_id="1"  name="Urinalysis"                          code="27171005"  creator="1"  date_created="2022-02-15 10:51:24"  retired="0" uuid="6633fdec-0954-446e-b5d7-0b3efaea2317"/>
+    <concept_reference_term  concept_reference_term_id="284136"  concept_source_id="1"  name="Rapid Plasma Reagin Test Procedure"  code="19869000"  creator="1"  date_created="2022-02-15 10:59:48"  retired="0" uuid="f3b6c722-3450-4ae9-99d0-77499d45dc25"/>
+    <concept_reference_term  concept_reference_term_id="284137"  concept_source_id="1"  name="Microscopy Procedure"                code="117259009" creator="1"  date_created="2022-02-15 12:02:40"  retired="0" uuid="e401c0f3-6681-46ae-9079-0b96a57addae"/>
+
+    <concept_reference_term  concept_reference_term_id="284138"  concept_source_id="1"  name="White Blood Cell Count"                            code="391558003" creator="1"  date_created="2022-03-07 11:23:07"  retired="0" uuid="fb5d8f13-ced8-4e2e-8e64-e1e62338d5ac"/>
```
Comment (slubwama) [inline api/src/main/resources/metadata/Custom_Concepts.xml]: Remove old Mapping of 767002

File: api/src/main/resources/metadata/Custom_Concepts.xml [hunk 2]
```diff
@@ -9079,7 +9178,34 @@
...
+    <concept_reference_term  concept_reference_term_id="284139"  concept_source_id="1"  name="Neutrophil count (procedure)"                      code="30630007"  creator="1"  date_created="2022-03-07 12:46:36"  retired="0" uuid="3477ca82-76cf-4632-9f42-88cf8823bef4"/>
+    <concept_reference_term  concept_reference_term_id="284140"  concept_source_id="1"  name="Neutrophil percent differential count (procedure)" code="56972008"  creator="1"  date_created="2022-03-07 13:00:33"  retired="0" uuid="c567fa2b-e1cd-473b-a2ce-c3c398bde961"/>
```
Comment (slubwama) [inline api/src/main/resources/metadata/Custom_Concepts.xml]: Verify that this mapping is for concept 1021 "LYMPHOCYTES".

Comment (Mwanje) [reply api/src/main/resources/metadata/Custom_Concepts.xml]: @slubwama this verified

**Comment by slubwama** [comment]
@Mwanje  resolve conlicts

**Comment by slubwama** [comment]
@Mwanje  is this still Work in progress or its ready for review

**Comment by Mwanje** [comment]
> @Mwanje is this still Work in progress or its ready for review

@slubwama this PR is ready for review

File: pom.xml [hunk 3]
```diff
@@ -89,7 +89,7 @@
 		<dataintegrityVersion>4.4.3</dataintegrityVersion>
 		<emrapiVersion>1.29.0</emrapiVersion>
 		<eventVersion>2.9.0</eventVersion>
-		<fhir2Version>1.2.2</fhir2Version>
+		<fhir2Version>1.2.1</fhir2Version>
```
Comment (slubwama) [inline pom.xml]: Change to 1.3

**Pull request merged**